.pipelines/templates/.builder-release-template.yaml

```diff
@@ -95,7 +95,6 @@ steps:
       pushd /tmp
       GO111MODULE=on go install cuelang.org/go/cmd/cue@latest
```

Comment:
do we need to install the cue module ?

Reply:
we need it for components.cue/validation of components.json right?